[2025-04-12T19:23:16.425+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-12T19:23:16.449+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pull_trade_data.stream_data_from_api manual__2025-04-12T19:23:12.745618+00:00 [queued]>
[2025-04-12T19:23:16.457+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pull_trade_data.stream_data_from_api manual__2025-04-12T19:23:12.745618+00:00 [queued]>
[2025-04-12T19:23:16.457+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-04-12T19:23:16.469+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): stream_data_from_api> on 2025-04-12 19:23:12.745618+00:00
[2025-04-12T19:23:16.483+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=802) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-12T19:23:16.488+0000] {standard_task_runner.py:72} INFO - Started process 803 to run task
[2025-04-12T19:23:16.494+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'pull_trade_data', 'stream_data_from_api', 'manual__2025-04-12T19:23:12.745618+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/testy_web_scoket.py', '--cfg-path', '/tmp/tmprqav9n6p']
[2025-04-12T19:23:16.519+0000] {standard_task_runner.py:105} INFO - Job 8: Subtask stream_data_from_api
[2025-04-12T19:23:16.653+0000] {task_command.py:467} INFO - Running <TaskInstance: pull_trade_data.stream_data_from_api manual__2025-04-12T19:23:12.745618+00:00 [running]> on host cf80ab16823f
[2025-04-12T19:23:16.922+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='kibret_og' AIRFLOW_CTX_DAG_ID='pull_trade_data' AIRFLOW_CTX_TASK_ID='stream_data_from_api' AIRFLOW_CTX_EXECUTION_DATE='2025-04-12T19:23:12.745618+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-12T19:23:12.745618+00:00'
[2025-04-12T19:23:16.927+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-12T19:23:17.349+0000] {_logging.py:89} INFO - Websocket connected
[2025-04-12T19:23:17.352+0000] {log.py:232} WARNING - [2025-04-12 19:23:17.352013] ### connection is open ###
[2025-04-12T19:23:51.655+0000] {logging_mixin.py:190} INFO - successful ingestion
[2025-04-12T19:23:51.715+0000] {_logging.py:77} ERROR - error from callback <bound method YLiveTicker.on_close of <yliveticker.YLiveTicker object at 0xffff787595e0>>: YLiveTicker.on_close() takes 2 positional arguments but 4 were given
[2025-04-12T19:23:51.719+0000] {log.py:232} WARNING - [2025-04-12 19:23:51.719002] YLiveTicker.on_close() takes 2 positional arguments but 4 were given
[2025-04-12T19:28:51.700+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-12T19:28:51.847+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-12T19:28:51.848+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=pull_trade_data, task_id=stream_data_from_api, run_id=manual__2025-04-12T19:23:12.745618+00:00, execution_date=20250412T192312, start_date=20250412T192316, end_date=20250412T192851
[2025-04-12T19:28:51.935+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-12T19:28:51.984+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-12T19:28:51.986+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
